app/main.py

```diff
@@ -1,2 +1,68 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity: int = 8
+        self.threshold: float = 0.66
+        self.size: int = 0
+        self.hash_table: list = [None] * 8
+
+    def __getitem__(self, key: Any) -> Any:
```

Comment:
Are u sure key could have any type? Dictionaries in Python require their keys to be **hashable**